.github/workflows/open-source-unit-tests.yml

```diff
@@ -0,0 +1,136 @@
+name: Py-marqo  open source unit tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      push_to:
```

Comment:
might be a misleading input name, as the image should already be pushed at this time. Maybe `pull_from` or `image_location`

Reply:
good catch!

Reply:
Changed to `image_registry_location`